pkg/services/libraryelements/libraryelements_get_test.go

```diff
@@ -69,6 +69,8 @@ func TestGetLibraryElement(t *testing.T) {
 				}
 			}
 
+			sc.reqContext.SignedInUser.Permissions[sc.reqContext.OrgID][dashboards.ActionFoldersRead] = []string{dashboards.ScopeFoldersAll}
```

Comment:
nit: could you add a comment about this. this is a bit unclear by just specifying permissions here. Now I understand why it is needed but if someone would come and read this, it might be unclear